.github/workflow/aggregate.yaml

```diff
@@ -40,3 +40,13 @@ jobs:
 
       - name: Publish output
         run: ardrive upload-file --local-path ./daily_output.json -F "d62a47c3-0b9d-4442-ac72-252a239d0469" -w /tmp/wallet --no-bundle --upsert --dry-run
+
+      - name: Read non unbundled bundles
```

Comment:
Where are we using this env variable?

Reply:
Nowhere. e4701eb127983b86021e035fa41f12e4b617e907